src/animation.js

```diff
@@ -197,6 +196,14 @@ class AnimationPlayer {
     this.task_ = this.vsync_.createAnimTask(this.contextNode_, {
       mutate: this.stepMutate_.bind(this),
     });
+
+
+    if (this.vsync_.canAnimate(this.contextNode_)) {
+      this.task_(this.state_);
+    } else {
+      dev().warn(TAG_, 'cannot animate');
+      this.complete_(/* success */ false, /* dir */ 0);
```

Comment:
Curious: isn't `complete_` also private state that should be removed?

Reply:
I don't understand your question? It's a private method called from multiple locations.

I'm only interested in removing private fields that are uncalled, or called from external locations (outside the class scope, meaning the field isn't private at all).

Reply:
Makes sense. Carry on.